src/content/docs/change-tracking/change-tracking-view-analyze.mdx

```diff
@@ -194,6 +196,28 @@ To add a custom trend board:
 5. Click **What do you want to track** and select a metric or event.
 6. Click **How do you want to aggregate that?** and select a function.
 
+### See the effects of changes on web transactions [#web-transactions]
+
+With change tracking, you can see details about how web transactions were affected by an APM application deployment. Below **Web transaction impacts**, you'll see a table that shows performance indicators for up to ten of the application's most time-consuming web transactions:
+
+<img
+    title="Screenshot showing where to view the impacts to web transactions"
+    alt="Screenshot showing where to view the impacts to web transactions"
+    src={trackingWebTransactionsImpact}
+/>
+
+To control what the table displays:
+
+* See how various metrics were affected for **This deployment** by selecting the **Metric** dropdown and choosing among these metrics:
+    * Throughput
+    * Total time
+    * Average time
+    * Apdex score
+    * Error rate
+* As you change the before-and-after time window in the table, keep in mind that if the time range after the deployment ends in the future, you may see incomplete transaction data.
+* To get the table to show a side-by-side comparison with another deployment, select another deployment in **compared with**.
+* If you hover over a value in the **Transaction name** column, you'll see a tooltip summarizing the performance for all five metrics for that transaction. The tooltip also has links to the APM transaction details so you can dig into detailed transaction-level data.
```

Comment:
i do wonder if explaining tool tips is useful, just because the idea is that once a NR user undrestand there are tool tips they'd look for them. so it may be that this is unnecessary. but like many things it's a judgment call I think. personally I'd remove it with the thought that: there are many tips/onramps in the UI and we shouldn't feel the need to replicate their work.